community-factory/Cargo.toml

```diff
@@ -19,3 +20,16 @@ lto = true
 debug = false
 panic = "abort"
 overflow-checks = true
+
+# fields to configure build with WASM reproducibility, according to specs  
```

Comment:
@petersalomonsen can you remove this section?
Until build scripts for subcontracts are properly integrated, docker builds won't work,
and you've also modified to `--no-docker` in `.github/workflows/release.yml`

Reply:
reproducible build sections are removed now @dj8yfo